sdk/storage/azure-storage-blob/azure/storage/blob/__init__.py

```diff
@@ -49,7 +50,10 @@
     ContainerSasPermissions,
     BlobSasPermissions,
     CustomerProvidedEncryptionKey,
-    ContainerEncryptionScope
+    ContainerEncryptionScope,
+    QuickQueryError,
```

Comment:
Can we rename both QuickQueryError and QuickQueryReader?

Reply:
what would you suggest?